inventory/all_projects/gitlab

```diff
@@ -0,0 +1,4 @@
+[gitlab_production]
+git.lib.princeton.edu
```

Comment:
I would love to see the VM names changed also - gitlab.lib.princeon.edu and gitlab-staging.lib.princeton.edu.

Reply:
does gitlab-stagingvm and gitlab-prodvm work

Reply:
Yep, see suggestions below.